### PR TITLE
Update email address for press

### PR DESCRIPTION
**Pull request by BraydenGirard**
Wording may need to be updated here and I am unsure what the proper email address should be for the comms team.

Resolves https://github.com/getsolus/solus-site.github.io/issues/45

### CODE REVIEW

File: content/press.html
Comment (davidjharder) [inline content/press.html:31]:
```suggestion
    Reach out to our communications team to ask about our products (remove NOSPAM):
```

**Review by davidjharder — CHANGES_REQUESTED** [review]
Thanks for the fix-up.
I've created the "comms" address. I prefer a simpler writing style.

**Comment by davidjharder** [comment]
Looks good! I'll merge this sometime later when I'm back home

**Review by davidjharder — APPROVED** [review]

**Comment by TraceyC77** [comment]
@BraydenGirard  Please update the commit message before you merge with
"Fixes #45"

**Comment by BraydenGirard** [comment]
@TraceyC77 is that what you meant ? I updated this PR name to match. I can not merge though as I do not have write access.

**Pull request merged**
